lib/rubocop/cop/style/indentation_width.rb

```diff
@@ -85,7 +85,8 @@ def on_send(node)
           return unless modifier_and_def_on_same_line?(receiver, method_name,
                                                        args)
 
-          _method_name, _args, body = *args.first
+          body = args.first.to_a.last
```

Comment:
Generally, I prefer us to use splatting. You can use `*_, body = *args.first` to achieve the same thing.